docker/Dockerfile

```diff
@@ -35,8 +31,12 @@ RUN adduser --disabled-password --gecos '' docker && \
 USER docker
 WORKDIR /home/docker
 
+# define variables
+ENV setools_repo https://github.com/kayrus/sedump
```

Comment:
I would like everything synced with TresysTechnology's setools if possible. Can you set setools_repo back to ``https://github.com/TresysTechnology/setools`` and use ``setools_rev`` to ``4.0.0-alpha3`` instead ?